README.rst

```diff
@@ -57,7 +57,7 @@ including a list of features.
 Requirements
 ============
 
-* Python 3.6+
+* Python 3.7+
```

Comment:
See also `docs/intro/install.rst` ([see how we changed it when dropping 3.5 support](https://github.com/scrapy/scrapy/pull/4742/files#diff-5967a18372589c0f3cdf68a1f2bddbdf069678675819eb828b1a6c9d3323238d))

Looking for “3.6” I also found references worth updating or removing in:

```
docs/contributing.rst
docs/topics/items.rst
tests/test_utils_python.py (we can probably simplify to keep only the statement within the if)
```